docs/source/blocks/block-style-wrapper.md

```diff
@@ -14,273 +14,150 @@ myst:
 ```{versionadded} 16.0.0
 ```
 
-The block style wrapper is part of a block anatomy.
-It allows you to inject styles from the block schema into the block wrapper in the form of class names.
-It wraps the block edit and the view components.
+The Style Wrapper is part of a block’s anatomy.
```

Comment:
I must have missed something.

1. Why are you now calling this thing "Style Wrapper" instead of "block style wrapper"?
2. Should it be a proper noun "Style Wrapper" or just a noun "style wrapper"? This is important for spellcheckers and consistency. This looks like a plain old noun to me, and as such does not need to be added to Vale, but "Style Wrapper" would need to be added to Vale.

Reply:
Also, whatever its correct spelling, we ought to have the term defined in the glossary in `plone/documentation`.

Also, also, do we need to change the title of this page?

Reply:
ok, changing.